### PR TITLE
(SIMP-8926) Bootstrap  needs absolute path

### PR DESCRIPTION
**Pull request by jeannegreulich**
- changed a call to puppet in the bootstrap_simp_client to
  use absolute path to puppet command.

SIMP-8926 #close

### CODE REVIEW

**Review by op-ct — APPROVED** [review]

File: files/var/www/ks/bootstrap_simp_client [hunk 1]
```diff
@@ -111,13 +111,13 @@ EOM
 
   def disable_service(service_name)
     info(title("Disabling #{service_name} service", 2))
-    result = execute("puppet resource service #{service_name} enable=false")
+    result = execute("/opt/puppetlabs/bin/puppet resource service #{service_name} enable=false")
```
Comment (lnemsick-simp) [inline files/var/www/ks/bootstrap_simp_client:114]: * Please make `/opt/puppetlabs/bin/puppet` a class constant (e.g., `PUPPET`) or variable (e.g. `@puppet`)
* The `if File.exists?('/opt/puppetlabs/bin/puppet` logic on lines 18-22 no longer make sense if you are assuming a specific path for the puppet executable.  Please fix.

**Review by lnemsick-simp — CHANGES_REQUESTED** [review]
Couple of nits

**Comment by lnemsick-simp** [comment]
Unit tests are failing in code that was not modified by this PR.

**Comment by lnemsick-simp** [comment]
I'll look into the acceptance tests failures.

**Comment by lnemsick-simp** [comment]
The problem is that the `domain` fact returns nil.  /etc/resolv.conf does not have a `search` or `domain` entry in it and `hostname` does not return a fqdn.  So the `domain` fact returns nil.

NOTE:  This is the contents of /etc/resolv.conf:
# Generated by NetworkManager
nameserver 8.8.8.8
options attempts:2 ndots:1 rotate timeout:2

**Review by lnemsick-simp — APPROVED** [review]

**Pull request merged**